### PR TITLE
Update our version of Loris to do color profile conversion

### PR DESCRIPTION
**Pull request by alexwlchan**
### What is this PR trying to achieve?

This updates our version of Loris to include https://github.com/loris-imageserver/loris/pull/350, which adds support for sRGB colour conversion in JPEGs. It also includes an sRGB colour profile and the required config.

Related: #734.

### Who is this change for?

People who want good-looking images! 🖼 

### Have the following been considered/are they needed?

- [ ] Deployed new versions
- [ ] Run `terraform apply`.

### CODE REVIEW

File: loris/Dockerfile [hunk 1]
```diff
@@ -3,16 +3,18 @@ FROM ubuntu:14.04
...
 RUN pip install awscli
 
-ENV LORIS_COMMIT 400bc3979a7dc3ff938abff84665c2731f750a69
+ENV LORIS_COMMIT 1e5de1f8b4dedcd80506c1f531fc76caec8ad504
 
 COPY install_loris.sh /install_loris.sh
 RUN /install_loris.sh
 
-RUN apt-get install -y uwsgi
+# This is an sRGB color profile downloaded from
```
Comment (kenoir) [inline loris/Dockerfile]: 👍

File: loris/Dockerfile [hunk 2]
```diff
@@ -3,16 +3,18 @@ FROM ubuntu:14.04
 RUN apt-get update
 
 RUN apt-get install -y python python-pip python-setuptools python-dev \
-    uwsgi-plugin-python
+    uwsgi uwsgi-plugin-python
```
Comment (kenoir) [inline loris/Dockerfile]: Why do we know include `uwsgi` by itself?

Comment (alexwlchan) [reply loris/Dockerfile]: We already did – this just puts our `apt-get install` commands on one line. Previously we were installing it after we’d installed Loris.

Comment (kenoir) [reply loris/Dockerfile]: Ah, I see it.

**Pull request merged**